src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -242,7 +242,7 @@ static void window_view_clipping_mouseup(rct_window* w, rct_widgetindex widgetIn
             _previousClipSelectionA = gClipSelectionA;
             _previousClipSelectionB = gClipSelectionB;
             gClipSelectionA = { 0, 0 };
-            gClipSelectionB = { MAXIMUM_MAP_SIZE_TECHNICAL - 1, MAXIMUM_MAP_SIZE_TECHNICAL - 1 };
+            gClipSelectionB = { MAXIMUM_MAP_SIZE_BIG - 1, MAXIMUM_MAP_SIZE_BIG - 1 };
```

Comment:
I don't think this is right. `gClipSelectionA` and `gClipSelectionB` are of type `CoordsXY`, but `MAXIMUM_MAP_SIZE_BIG` is for big coords, right?

Reply:
CoordsXY **are** big coords.